docs-content/index.html.md.erb

```diff
@@ -75,40 +74,47 @@ The following table provides version and version-support information about Cyber
 
 CyberArk Conjur Service Broker for PCF has the following requirements:
 
-+ You must have an existing Conjur appliance installed.  The appliance may be external to the PCF environment. Supported versions are:
-  * Enterprise Conjur (v4.9.12.0 and later)
++ You must have an existing Conjur instance installed.  The Conjur instance may be external to the PCF environment. Supported versions are:
+  * Conjur Enterprise (v4.9.12.0 and later)
       * Use v4.9.17.0 or later to search for CF hosts in the Conjur UI
-  * Open Source Conjur (v5.x)
+  * Conjur Open Source (v1.0 and later)
       * Hosted Conjur is a hosted version of Open Source Conjur. You can quickly get a hosted Conjur instance running for evaluation purposes. Visit the [hosted Conjur page](https://www.conjur.org/get-started/try-conjur.html) to sign up.
 
-+ To configure the PCF tile, you must know the following information about your Conjur appliance:
-  * Conjur account, which is specified when the Conjur appliance is created and configured. For hosted Conjur, the account is usually an email address that you specify on the web form.
-  * URL to the Conjur appliance.
++ To configure the PCF tile, you must know the following information about your Conjur installation:
+  * Conjur account, which is specified when Conjur is installed and configured. For hosted Conjur, the account is usually an email address that you specify on the web form.
```

Comment:
I still think it'd be nice to organize this information better - I wonder if a table would work?

Reply:
I put the information in table. Let me know what you think!